No rule to make target `locale/gpxlab_fi.qm'

Hello,

I'm trying to compile GPXLab with OS X 10.13.16

I did

```bash
$ git clone https://github.com/BourgeoisLab/GPXLab
$ cd GPXLab
$ qmake
$ make
cd QMapControl/ && ( test -e Makefile || /Users/scls/opt/anaconda3/bin/qmake -o Makefile /Users/scls/github/others/GPXLab/QMapControl/QMapControl.pro ) && /Applications/Xcode.app/Contents/Developer/usr/bin/make -f Makefile
make[1]: Nothing to be done for `first'.
cd GPXLab/ && ( test -e Makefile || /Users/scls/opt/anaconda3/bin/qmake -o Makefile /Users/scls/github/others/GPXLab/GPXLab/GPXLab.pro ) && /Applications/Xcode.app/Contents/Developer/usr/bin/make -f Makefile
make[1]: *** No rule to make target `locale/gpxlab_fi.qm', needed by `GPXLab.app/Contents/Resources/translations/gpxlab_fi.qm'.  Stop.
make: *** [sub-GPXLab-make_first-ordered] Error 2
 $ open GPXLab/GPXLab.app
The application cannot be opened because its executable is missing.
```

Any idea?

Kind regards

--- a/README.md
+++ b/README.md
@@ -43,10 +43,25 @@ License
 -------
 This software is free software and licensed under GPL version 3. 
 
-Compiling
----------
+Development
+-----------
 The project is separated in two sub-projects GPXLab and QMapControl. QMapControl is a widget to display a map. Since I had to made some modifications to the original code I added the modified version here. QMapControl is compiled as a library and linked to the main sub-project GPXLab.
- 
+
+Get latest source from master branch using:
+
+```bash
+$ git clone https://github.com/BourgeoisLab/GPXLab/
+```
+
+Compile source using:
+
+```bash
+$ cd GPXLab
+$ lrelease GPXLab/GPXLab.pro
+$ qmake
+$ make
+```
+
 Credits
 -------
 - **QCustomPlot Qt Widget:** http://www.qcustomplot.com

